CODEOWNERS

```diff
@@ -119,4 +119,5 @@ packages/shared_preferences/shared_preferences_windows/** @cbracken
 packages/url_launcher/url_launcher_windows/**            @cbracken
 
 # - DevTools extensions
-packages/shared_preferences/shared_preferences_tools/**  @tarrinneal
+# @adsonpleal is the actual mantainer of shared_preferences_tool
```

Comment:
@kenzieschmoll and @stuartmorgan is this enough?

Reply:
Looks good to me, but I will let @stuartmorgan confirm. Nit: typo in "maintainer"

Reply:
Let's add "but is not yet a committer, so can't be listed as the owner."